lib/pages/setting/setting_screen.dart

```diff
@@ -0,0 +1,39 @@
+import 'widgets/avatar.dart';
+import 'widgets/birthday_date_picker.dart';
+import 'package:flutter/material.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'widgets/gender_radio.dart';
+import 'widgets/language_dropdown.dart';
+
+class SettingScreen extends StatefulWidget {
+  final User user;
+
+  SettingScreen(this.user);
+
+  @override
+  _SettingState createState() => _SettingState();
+}
+
+class _SettingState extends State<SettingScreen> {
+  User get user => widget.user;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
```

Comment:
```suggestion
    return Column(
        mainAxisAlignment: MainAxisAlignment.center,
        mainAxisSize: MainAxisSize.max
```
try this and reply me if it work

Reply:
It's work 👍